src/main/java/hudson/tasks/junit/CaseResult.java

```diff
@@ -397,26 +404,52 @@ public int getPassCount() {
      */
     @Exported(visibility=9)
     public int getFailedSince() {
-        // If we haven't calculated failedSince yet, and we should,
-        // do it now.
+        // If we haven't calculated failedSince yet, and we should, do it now.
         if (failedSince==0 && getFailCount()==1) {
             CaseResult prev = getPreviousResult();
-            if(prev!=null && !prev.isPassed())
+            if(prev!=null && !prev.isPassed()) {
                 this.failedSince = prev.getFailedSince();
+            }
             else if (getRun() != null) {
                 this.failedSince = getRun().getNumber();
             } else {
                 LOGGER.warning("trouble calculating getFailedSince. We've got prev, but no owner.");
-                // failedSince will be 0, which isn't correct. 
+                // failedSince will be 0, which isn't correct.
             }
         }
         return failedSince;
     }
-    
+
+    /**
+     * If this test was skipped, then return the build number
+     * when this test was first skipped.
+     */
+    @Exported(visibility=9)
```

Comment:
Shouldn't the visibility of the skipped tests be higher than the visibility of failed tests?

Reply:
On second thought I think the visibility for getFailedSince and getSkippedSince should be identical.